src/components/Navbar/styles.scss

```diff
@@ -17,6 +17,10 @@
   padding-right: $tw-sz-base-small;
   padding-top: $tw-sz-base-small;
 
+  &.transparent {
+    background-color: transparent !important;
```

Comment:
made this `!important` because during testing with terraware-web I couldn't figure out how to otherwise make sure it always overrides the assigned background. Seems okay perhaps, since transparency here is a very intentional flag to set?

Reply:
agree, this makes sense here.